kcidb/cache/__init__.py

```diff
@@ -51,7 +51,7 @@ def store(self, url):
         object_name = self._format_object_name(url)
 
         # Cache every 4096th URL only for the trial period
```

Comment:
Looks good, but please update the comment as well.

Reply:
Yep!

Reply:
done